doc/modules/calibration.rst

```diff
@@ -84,6 +84,16 @@ typical for maximum-margin methods (compare Niculescu-Mizil and Caruana [1]_),
 which focus on difficult to classify samples that are close to the decision
 boundary (the support vectors).
 
+Confidence intervals for the fraction of positives in each bin can be drawn
+using standard deviation estimates by setting `return_std=True` in
+:func:`calibration_curve`. The standard deviation of the fraction of positives
+for each bin is estimated with the formula below, where :math:`\hat{p}` is the
+fraction of positives in the bin, and :math:`n` is the number of samples in the
+bin:
+
+.. math::
+       \sqrt{ \frac{ \hat{p} (1 - \hat{p}) }{ n } }
```

Comment:
I don't get this. The empirical variance of the binomial distribution is `p (1 - p)`. So the standard deviation is `sqrt(p (1 - p))`.

What you compute here seems to be the standard error instead. Which makes sens as you want to derive a confidence interval which is not possible from the standard deviation directly.

But then we have a naming issue because `return_std` means standard deviation in other places of scikit-learn, not standard error.

Reply:
I'll correct it to standard error, e.g., `return_std` -> `return_se`. (I originally called it standard error and then changed everything to standard deviation b/c I thought that was what people were more used to hearing.) Sorry about the incorrect naming.